src/zmq/zmqrpc.cpp

```diff
@@ -20,7 +20,7 @@ UniValue getzmqnotifications(const JSONRPCRequest& request)
             "\nReturns information about the active ZeroMQ notifications.\n"
             "\nResult:\n"
             "[\n"
-            "  {                        (json object)\n"
+            "  {                        (object)\n"
```

Comment:
Half of the changes seem to be of this fashion. Mind to create a primitive scripted-diff commit in a separate pull request to get the easy to review things out of the way?

Reply:
Damn, they need to make a `sed`-builder plugin alongside find-and-replace in vscode, that supports negation. Way easier workflow for newer contributors than all the nuanced escape characters that are necessary in *nix tools.

Is it preferred to use `git grep (-l) (-L)` or `find . x` in scripted diffs?

Also, does this really make it easier to review for correctness? I'm not so sure.

Next commit will be a scripted diff for:
> Replace `(boolean)W` with `(boolean) W` in `.`

`git grep -l "(boolean)W" | xargs sed -i "s/(boolean)W/(boolean) W/g"`
is about right I think?

What if I'm stuck in PowerShell! No `xargs` available.

Reply:
And on Mac, `gsed` is needed to replicate `sed` behavior performed in scripted-diffs

Reply:
I'd do it like this: `sed -i --regexp-extended -e 's/\(json (array|object)\)/(\1)/g' $(git grep -l '(json ')`. There might be a hack to make it work on macOS, but we mostly care that it runs on Ubuntu et al.

Reply:
Pushed to https://github.com/ch4ot1c/bitcoin/tree/fix/rpc-descriptions, but the commit isn't appearing here. Strange...